node/tests/SharedTests.ts

```diff
@@ -4482,6 +4482,54 @@ export function runBaseTests<Context>(config: {
         config.timeout,
     );
 
+    it.each([ProtocolVersion.RESP2, ProtocolVersion.RESP3])(
+        "copy test_%p",
+        async (protocol) => {
+            await runTest(async (client: BaseClient, cluster: RedisCluster) => {
+                if (cluster.checkIfServerVersionLessThan("6.2.0")) return;
+
+                const source = `{key}-${uuidv4()}`;
+                const destination = `{key}-${uuidv4()}`;
+                const value1 = uuidv4();
+                const value2 = uuidv4();
+
+                // neither key exists
+                checkSimple(
```

Comment:
Same, can we use `expect` instead of `checkSimple` for the non-string returns?

Reply:
If I use expect, the check will fail. That's why I use checkSimple instead.